wiremock/delius/ndelius-casedata.json

```diff
@@ -2,7 +2,7 @@
   "mappings": [
     {
       "request": {
-        "url": "/delius-api/case-details/NOTFOUND",
```

Comment:
Are we sure we want to drop this part of the path? it means it won't be super obvious what API we are mocking in a wiremock request/call. It's probably not a huge issue on this service because we've not mocked a bunch of different services _currently_, but think it's good to keep it for future.

Reply:
We've renamed the container from `wiremock` to `delius`. The intention here is, similar to UPW, to spin up separate wiremock containers for each mocked API. This allows us to set up the container dependencies correctly.

For example:

```
delius:
  image: wiremock

san-api:
  image: wiremock
  
sp-api:
  depends_on:
    - delius
  environment:
    DELIUS_URL: http://delius:8080

coordinator-api:
  depends_on:
    - san-api
  environment:
    SAN_URL: http://san-api:8080
```